docs/source/pages/DD2090.rst

```diff
@@ -46,6 +46,17 @@ But if you want to enable :ref:`Hardware message syncing` for all 4 cameras, you
     with dai.Device(config) as device:
         device.startPipeline(pipeline)
 
+Syncing with OAK-FFC-IR
+-----------------------
+
+If you have an `OAK-FFC-IR <https://shop.luxonis.com/collections/early-access/products/oak-ffc-ir>`__ compatible board (rev. R5M1E5 and up) and wish to use the IR module, you **must** first configure from which camera you want to have STROBE signal sent to the connector on the left (to which you connect the IR module). 
+
+
+.. image:: /_static/images/FFC-4P/SW1.png
+
+If you have stereo pair on CAM_B and CAM_C, and want to have dot projector synced with stereo pair, you should move the switch 2 or 3 up (ON).
```

Comment:
AFAIK only 1 switch should be on - in this case, either 2/3 would work, as both left/right cameras have same strobe output.